test/test_interpolations.jl

```diff
@@ -252,6 +249,251 @@ using Ferrite: reference_shape_value, reference_shape_gradient
+    # 1) Unit property: ∫(Nⱼ ⋅ v f(s) dS) = 1/length(𝔇) ∀ ∈ 𝔇
+    #    Must hold for
+    #    length(𝔇) ≥ 1: f(s) = 1
+    #    length(𝔇) = 2: f(s) = 1 - s or f(s) = s for 1st and 2nd dof, respectively.
+    #    Additionally, should be zero for
+    #    length(𝔇) = 2: f(s) = s or f(s) = 1 - s for 1st and 2nd dof, respectively.
+    #    s is the path parameter ∈[0,1] along the positive direction of the path.
+    # 2) Zero along other edges: Nⱼ ⋅ v = 0 if j∉𝔇
+    @testset "H(curl) on RefCell" begin
+        lineqr = QuadratureRule{RefLine}(20)
+        for ip in (Nedelec{2, RefTriangle, 1}(), Nedelec{2, RefTriangle, 2}(), Nedelec{3, RefTetrahedron, 1}(), Nedelec{3, RefHexahedron, 1}())
+            cell = reference_cell(getrefshape(ip))
+            edges = Ferrite.edges(cell)
+            dofs = Ferrite.edgedof_interior_indices(ip)
+            x = Ferrite.reference_coordinates(geometric_interpolation(typeof(cell)))
+            @testset "$(getrefshape(ip)), order=$(Ferrite.getorder(ip))" begin
+                for (edge_nr, (i1, i2)) in enumerate(edges)
+                    Δx = x[i2] - x[i1]
+                    x0 = (x[i1] + x[i2]) / 2
+                    L = norm(Δx)
+                    v = Δx / L
+                    for (idof, shape_nr) in enumerate(dofs[edge_nr])
+                        nedgedofs = length(dofs[edge_nr])
+                        f(x) = nedgedofs == 1 ? 1.0 : (idof == 1 ? 1 - x : x)
+                        s = line_integral(lineqr, ip, shape_nr, x0, Δx, L, v, f)
+                        @test s ≈ one(s) / nedgedofs
+                        if nedgedofs == 2
+                            g(x) = idof == 1 ? x : 1 - x
+                            @test 1 ≈ 1 + line_integral(lineqr, ip, shape_nr, x0, Δx, L, v, g)
+                        end
+                    end
+                    for (j_edge, shape_nrs) in enumerate(dofs)
+                        j_edge == edge_nr && continue
+                        for shape_nr in shape_nrs
+                            for ξ in (x[i1] + r * Δx for r in [0.0, rand(3)..., 1.0])
+                                @test abs(reference_shape_value(ip, ξ, shape_nr) ⋅ v) < eps() * 100
+                            end
+                        end
+                    end
+                end
+            end
+        end
+    end
+
+    # Required properties of shape value Nⱼ of an edge-elements (Hdiv) on an edge with normal n, length L, and dofs ∈ 𝔇
+    # 1) Unit property: ∫(Nⱼ ⋅ n f(s) dS) = 1/length(𝔇) ∀ j ∈ 𝔇
+    #    Must hold for
+    #    length(𝔇) ≥ 1: f(s) = 1
+    #    length(𝔇) = 2: f(s) = 1 - s or f(s) = s for 1st and 2nd dof, respectively.
+    #    Additionally, should be zero for
+    #    length(𝔇) = 2: f(s) = s or f(s) = 1 - s for 1st and 2nd dof, respectively.
+    #    s is the path parameter ∈[0,1] along the positive direction of the path.
+    # 2) Zero normal component on other edges: Nⱼ ⋅ n = 0 if j∉𝔇
```

Comment:
Associated test definitions for [comment above](https://github.com/Ferrite-FEM/Ferrite.jl/pull/798/files#r1854906581) for Hdiv